e2eTests/e2eTestHelper.psm1

```diff
@@ -371,9 +385,31 @@ function CreateAlGoRepository {
     
     $tempRepoPath = Join-Path $tempPath ([GUID]::NewGuid().ToString())
     Expand-Archive -Path $zipFileName -DestinationPath $tempRepoPath
-    Copy-Item (Join-Path (Get-Item "$tempRepoPath\*").FullName '*') -Destination . -Recurse -Force
+    Copy-Item (Join-Path (Get-Item "$tempRepoPath/*/$templateFolder").FullName '*') -Destination . -Recurse -Force
     Remove-Item -Path $tempRepoPath -Force -Recurse
     Remove-Item -Path $zipFileName -Force
+    if ($templateFolder) {
+        # This is a direct AL-Go development repository
+        # Replace URL's + references to microsoft/AL-Go-Actions with $templateOwner/AL-Go/Actions
+        Get-ChildItem -Path . -File -Recurse | ForEach-Object {
+            $file = $_.FullName
+            $lines = ([string](Get-Content -Encoding UTF8 -Raw -path $file)).Replace("`r","").Split("`n")

```

Comment:
The issue here, I think is Linux (LF) vs. Windows (CRLF) line endings.
Files might be written in one format and read on the other - which have caused issues in earlier versions of AL-Go for GitHub.
This construct is used successfully for Update AL-Go System files to avoid detecting changes when run on different OS' in Update AL-Go System Files - haven't tried to use Get-Content without the -raw here